demos/sd-turbo/index.js

```diff
@@ -942,6 +953,23 @@ const ui = async () => {
             break;
     }
 
+    if (config.devicetype.toLowerCase().indexOf("cpu") > -1 || config.provider.toLowerCase().indexOf("wasm") > -1) {
```

Comment:
Other places use `deviceType` as camelCase (e.g. `if (deviceType.toLowerCase() === "npu")`). Should this too?